Networking/Networking/Mapper/JetpackConnectionURLMapper.swift

```
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Mapper: Jetpack Connection URL
+///
+struct JetpackConnectionURLMapper: Mapper {
+
+    /// (Attempts) to convert the response into a URL.
+    ///
+    func map(response: Data) throws -> URL? {
```

What do you think about throwing an error instead of returning an optional URL? (As the method is already following that pattern) 

Also, we could follow the same pattern in other places like the completion blocks of `JetpackConnectionAction`, `JetpackConnectionRemote` and `JetpackConnectionStore`. (i.e. Send Error instead of optional URL)

---

Good point! Updated in fc69543e61e6bf8116e3dd531f674e7960ae6f5c.